src/notification-banner/private/useForceUpdate.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
Is this generic enough that we should put it somewhere else?

Reply:
Most def, probably not right now tho